homeassistant/components/vicare/sensor.py

```diff
@@ -229,7 +229,7 @@
     SENSOR_POWER_PRODUCTION_CURRENT: {
         CONF_NAME: "Power production current",
         CONF_ICON: None,
-        CONF_UNIT_OF_MEASUREMENT: ENERGY_WATT_HOUR,
+        CONF_UNIT_OF_MEASUREMENT: POWER_WATT,
         CONF_GETTER: lambda api: api.getPowerProductionCurrent(),
         CONF_DEVICE_CLASS: DEVICE_CLASS_ENERGY,
```

Comment:
Sorry, I forgot. We should change the device class too. Please change to `DEVICE_CLASS_POWER`.

Reply:
Please make a new PR.

Reply:
https://github.com/home-assistant/core/pull/49040